scipy/linalg/tests/test_batch.py

```diff
@@ -45,9 +47,9 @@ def batch_test(self, fun, arrays, *, core_dim=2, n_out=1, kwargs=None, dtype=Non
         arrays = (arrays,) if not isinstance(arrays, tuple) else arrays
 
         # Identical results when passing argument by keyword or position
-        res2 = fun(*arrays, **kwargs)
+        res2 = fun(*arrays, **copy.deepcopy(kwargs))
```

Comment:
Rather than the `deepcopy`s, what do you think of passing an integer for `rng`? The `rng` fixture is just for convenience.

Reply:
Oh yeah that would be more straightforward